INN/umbrella-citylimits
# Create article template, style header

## Changes

This pull request makes the following changes:

- Replaces Largo's two-column template with this modified one, and
    - [x] styles for desktop
    - [x] styles for mobile
    - [x] byline handling of multiple authors?
    - [x] correct social media buttons, from https://github.com/INN/umbrella-citylimits/pull/66
- ignores Largo's subtitle function, but makes a note about styling it in the future

## Why



For #50 

## Testing/Questions

Features that this PR affects:

- pages with the two-column template
- articles with the two-column template


Questions that need to be answered before merging:

- [x] what was the markup @joshdarby used for the social media buttons?
- [ ] Where did the email icon in the mockups come from?
- [ ] are we worried about the floating social buttons?

Steps to test this PR:

1. check out an article page
2. check out a page-page.
